src/olympia/addons/indexers.py

```diff
@@ -310,6 +317,11 @@ def extract_document(cls, obj):
         ]
 
         data['is_featured'] = obj.is_featured(None, None)
+        data['featured_for'] = [
+            {'application': [app], 'locales': [
+                # ES doesn't like None values.
+                locale if locale else 'NONE' for locale in locales]}
```

Comment:
You could use https://www.elastic.co/guide/en/elasticsearch/reference/current/null-value.html to avoid doing that yourself when indexing (by default, `null` values are the same thing as empty in ES, but with `null_value` you can change that behavior at the mapping level)

@EnTeQuAk do you think it's worth doing here?

Reply:
`null_value` looks clean to me so should be worth doing.

Reply:
hmm, on second thought why can't we just ignore null values in that list?

Reply:
`null` means the collection isn't targeted to a specific locale, so should be shown to all.

Reply:
https://github.com/mozilla/addons-server/pull/6339/commits/cc02a18364a3485fda2cdaa17e4763bf51e10b04